docs/swarm-api.md

```diff
@@ -24,71 +27,157 @@ POST "/images/create" : "docker import" flow not implement
 
 ## Endpoints which behave differently
 
-* `GET "/containers/{name:.*}/json"`: New field `Node` added:
+<table>
```

Comment:
Okay just confirming what this table was supposed to have. I think it would take a careful look at the new Docker API to complete this table. I'll keep you updated as I find more stuff. For now, you can add volume filters (`dangling=true`) if possible.